images/rag/requirements.txt

```diff
@@ -1,11 +1,11 @@
-llama_index
+llama_index==0.9.39
```

Comment:
any reason for picking this version?

Reply:
It's the same as what lightspeed-service is using. Without it, the latest, 0.10.x, gets pulled in and they rearranged some of the module structure.